modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -256,8 +259,9 @@ const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets, clipbo
   });
 
   const tableCellBorderWidthsList = getTableBorderWidths(editor);
-  editor.ui.registry.addNestedMenuItem('tablecellvalign', {
+  editor.ui.registry.addNestedMenuItem('tablecellborderwidth', {
     icon: 'vertical-align',
```

Comment:
The icon is still `vertical-align`? Here and the `tablecellborderstyle` one too

Reply:
- [ ] Fix incorrect icons